docs/markdown/Dependencies.md

```diff
@@ -173,6 +173,10 @@ it automatically.
     cmake_dep = dependency('ZLIB', method : 'cmake', modules : ['ZLIB::ZLIB'])
 ```
 
+Support for adding additional `COMPONENTS` for the CMake `find_package` lookup
+is provided with the `components` kwarg (*introduced in 0.54.0*). All specified
+componets will be passed directly to `find_package(COMPONENTS)`.
+
```

Comment:
In the past we've mapped this to the existing `modules` arugment instead of `components`, even if cases (like llvm-config) when `components` is what the underlying tool calls them. I'd like to keep that so we don't have two arguments that do the same thing

Reply:
I agree that this makes semantically more sense, however, we are currently using `modules` already for selecting the imported CMake targets (think `Zlib::Zlib` or `KF5::KTextEditor`). Changing the meaning of the `modules` kwarg would be a breaking change (possible to do with a deprecation period and a concrete plan, but out of the scope of this PR).

If I were to write the CMake backend from scratch, I would also use `modules` for `COMPONENTS` and add an extra kwarg for CMake targets.

Reply:
ahhhhh. Okay, that makes sense. We can leave this as-is then